### PR TITLE
Update onboarding design

### LINKED ISSUE #228: Updating the  on-boarding
We are updating the onboarding screens. 

1_Adding color backgrounds
2_Change first texts, adding a hyperlink to faq 

_Design material: in **Yellow color**_ https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/circles-myxogastria?node-id=3820%3A19832

### PR DESCRIPTION
**Pull request by mikozet**
This PR updates onboarding process according to design

### CODE REVIEW

File: src/hooks/header.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { useScrollTrigger } from '@material-ui/core';
```
Comment (adzialocha) [inline src/hooks/header.js]: Maybe this file could be named `scrollTrigger.js` or similar?

File: src/styles/theme.js [hunk 2]
```diff
@@ -187,6 +188,8 @@ export default createMuiTheme({
         transition: 'color .01s',
       },
     },
+
+    // box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
```
Comment (adzialocha) [inline src/styles/theme.js]: Looks like something made its way in here :detective:

File: src/components/Tutorial.js [hunk 3]
```diff
@@ -27,12 +31,22 @@ const useStyles = makeStyles((theme) => ({
...
   tutorialSkipButton: {
     fontWeight: theme.typography.fontWeightMedium,
     textTransform: 'uppercase',
     marginRight: -3,
   },
+  tutorialHeader: {
+    background: 'transparent',
+  },
+  tutorialHeaderWithScroll: {
+    background: `${theme.palette.common.white}`,
```
Comment (adzialocha) [inline src/components/Tutorial.js]: No backticks are needed here.

File: src/components/Tutorial.js [hunk 4]
```diff
@@ -27,12 +31,22 @@ const useStyles = makeStyles((theme) => ({
...
+    transition: `${theme.transitions.create(['background'], {
```
Comment (adzialocha) [inline src/components/Tutorial.js]: Same here?

File: src/components/Tutorial.js [hunk 5]
```diff
@@ -1,24 +1,28 @@
...
 
 const useStyles = makeStyles((theme) => ({
   tutorialMobileStepper: {
     flexGrow: 1,
+    background: 'transparent',
   },
   tutorialMobileStepperDot: {
-    backgroundColor: 'transparent',
-    border: `1px solid ${theme.palette.text.primary}`,
+    backgroundColor: `${theme.palette.common.white}`,
```
Comment (adzialocha) [inline src/components/Tutorial.js]: No backticks needed here :+1:

File: src/components/BackgroundCurved.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+const useStyles = makeStyles((theme) => ({
+  wrapper: {
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    bottom: '50%',
+    zIndex: 0,
+    top: 0,
+    margin: '0 auto',
+    background: 'linear-gradient(180deg, #48B2B7 0%, #06FC9D 100%)',
```
Comment (adzialocha) [inline src/components/BackgroundCurved.js]: Could be cool to get the colors from somewhere where they have been defined, probably here: https://github.com/CirclesUBI/circles-myxogastria/blob/main/src/styles/theme.js#L30

**Review by adzialocha — COMMENTED** [review]
Yay! This looks really good! :sparkles:  I've just added some smaller notes here, but only details.

File: src/components/BackgroundCurved.js (hunk 6 above)
Comment (adzialocha) [reply src/components/BackgroundCurved.js]: Similar actually with the `zIndex`, its nice when they are all defined in one place.

**Comment by louilinn** [comment]
Just to be clear, we will not merge any visual updates into preview until current updates from Milestone "Summer updates" are merged into main.

**Review by louilinn — CHANGES_REQUESTED** [review]
Nicely done fixing @adzialocha comments @mikozet :star: 
I think the code looks good.

However looking at the UI I think we have some work left. I will make a comment with some screen shots. Basically the radius of the circles seems like it should be constant.

**Comment by louilinn** [comment]
I am not sure how to fix this exactly.. but some screenshots for reference:
Also I think the color is a bit too light? Or the fading starts too early in the top?

Screenshot from 2021-12-23 13-17-14
Screenshot from 2021-12-23 13-17-37
Screenshot from 2021-12-23 13-17-56
Screenshot from 2021-12-23 13-21-40

**Comment by mikozet** [comment]
> I am not sure how to fix this exactly.. but some screenshots for reference: Also I think the color is a bit too light? Or the fading starts too early in the top?
> 

I correct it
Zrzut ekranu 2022-01-7 o 13 55 42

**Comment by mikozet** [comment]
Update with commit which solves as well large desktop screens

**Comment by louilinn** [comment]
@mikozet this is an outdated PR right? If not you can reopen

**Pull request closed without merging**